packages/server/tests/acceptance/rateLimiter.spec.ts

```diff
@@ -158,4 +158,29 @@ describe('@ratelimiter Rate Limiters Acceptance Tests', function () {
       });
     });
   });
+
+  describe('Koa Server Timeout', () => {
```

Comment:
Either we can cancel the `server-config` CI and have this test here, or we keep the `server-config` CI and remove this test in ratelimiter.

Reply:
Agreed.  That got missed in the back and forth.  Removing.